ReactWindows/ReactNative.Common/ReactNative.Common.csproj

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I think as we alluded to on Discord, it would be better to keep the number of projects small, and best to just merge this with the Bridge project (obviously giving that project a new name).  Sorry for the flip flopping.